google-home-community.groovy

```diff
@@ -2358,6 +2554,16 @@ private traitFromSettings_ColorSetting(traitName) {
     return deviceTrait
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private traitFromSettings_Dock(traitName) {
+    return [
+        dockAttribute:settings."${traitName}.dockAttribute",
+        dockValue:             settings."${traitName}.dockValue",
+        dockCommand:           settings."${traitName}.dockCommand",
+        commands:              ["Dock"]
+    ]
```

Comment:
Looks like CodeFactor missed this one, but the formatting here is a bit off.  The general rule for this project is for maps to have one space between the longest key and its value and to align the other values with that.  For this one it would be something like this:
```suggestion
    return [
        dockAttribute: settings."${traitName}.dockAttribute",
        dockValue:     settings."${traitName}.dockValue",
        dockCommand:   settings."${traitName}.dockCommand",
        commands:      ["Dock"]
    ]
```